code/obj/item/gun/kinetic.dm

```diff
@@ -1392,6 +1392,47 @@ ABSTRACT_TYPE(/obj/item/gun/kinetic)
 			src.UpdateIcon()
 			return
 
+/obj/item/gun/kinetic/mrls
+	desc = "A multiple rocket launch system."
+	name = "\improper MRLS-6"
+	icon = 'icons/obj/large/64x32.dmi'
+	inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+	icon_state = "rpg7"
+	item_state = "rpg7"
+	wear_image_icon = 'icons/mob/clothing/back.dmi'
+	flags = ONBACK
+	w_class = W_CLASS_BULKY
+	throw_speed = 2
+	throw_range = 4
+	force = MELEE_DMG_LARGE
+	contraband = 8
+	ammo_cats = list(AMMO_ROCKET_ALL)
```

Comment:
we probably *don't* want people to be able to load this with 6 full-strength RPG rockets (and then rapid-fire them etc)